docs/welcome.md

```diff
@@ -1,52 +1,51 @@
 ---
 id: welcome
-title: Getting Started
+title: Get Started
 description: Master Amplication with in-depth developer guides, API details, and tutorials.
 sidebar_label: Introduction
 slug: /
 ---
 
 # Get Started with Amplication
 
-Hello and welcome to Amplication's developer documentation.
+Welcome to Amplication's developer documentation!
 
-[Amplication](/about/) is an open-source backend development tool. It accelerates the development of Node.js and .NET applications by auto-generating production-ready backends.
-
-Streamline your development process and increase productivity so you can focus on what matters most: **creating awesome applications**.
+Amplication is an open-source, AI-powered backend development platform that accelerates the creation of production-ready Node.js and .NET applications. With Amplication, you can focus on your core business logic while we handle the boilerplate.
 
 ## Quick Start
 
-[Create your first service](/first-service/) with Amplication.
-Set up a new service in less than 1 minute with our onboarding wizard.
-
-## Key Features of Amplication
-
-Amplication helps you build a fully functional backend in minutes. Just define your [data models](/how-to/create-entity/), roles, and [permissions](/how-to/set-access-permissions/) — and Amplication will generate a Node.js or .NET application with everything you need.
-
-All generated applications include the following features:
+Get up and running in minutes:
 
-- Fully-functional [REST](/api/#rest-api) and [GraphQL](/api/#graphql-apis) APIs for all your data models.
-- [Authentication](/authentication/) & Authorization
-- React Admin UI
-- Logging
-- [Smart Git Sync](/smart-git-sync/)
-- [Automatic Code Generation](/getting-started/view-generated-code/)
-- [Deploy](/deploy/) & Publish
+1. [Create your first service](/first-service/)
+2. [Set up entities and fields](/set-up-entities/)
+3. [Configure roles and permissions](/configure-roles-and-permissions/)
+4. [Commit changes and build](/commit-and-build-new-versions/)
 
-In addition, developers can utilize their skills to [customize the generated application](/how-to/custom-code/) afterwards.
+## Key Features
```

Comment:
Maybe we need to mention also import db schema- which is relevant for existing services to modernize easily using Amplication